src/lib/Queue.js

```diff
@@ -12,7 +12,7 @@ class Queue {
     if (!name) {
       // If name is not explicitly set, generate Queue name from class name:
       // - Removes conventional Q at the end
-      // - Parametrizes string
+      // - Param cases the string
```

Comment:
Should this be "parameterizes"?

```suggestion
      // - Parameterizes the string
```